kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/events/EventContext.kt

```diff
@@ -78,4 +82,30 @@ public open class EventContext<T : Event>(
         key: String,
         replacements: Array<Any?> = arrayOf()
     ): String = translate(key, eventHandler.extension.bundle, replacements)
+
+    /**
+     * Given a translation key and possible replacements,return the translation for the given locale in the
+     * extension's configured bundle, for the locale provided by the bot's configured locale resolvers.
+     */
+    public suspend fun translate(
+        key: String,
+        replacements: Map<String, Any?>
+    ): String = translate(key, eventHandler.extension.bundle, replacements)
+
+    private suspend fun resolveLocale(): Locale? {
+        val eventObj = event as Event
+
+        val guild = guildFor(eventObj)
+        val channel = channelFor(eventObj)
+        val user = userFor(eventObj)
+
+        for (resolver in eventHandler.extension.bot.settings.i18nBuilder.localeResolvers) {
+            val result = resolver(guild, channel, user, interactionFor(eventObj))
+
+            if (result != null) {
+                return result
+            }
+        }
+        return null
```

Comment:
You've removed a newline here.

Reply:
Fixed this, but it didn't update in here for some reason, But if you check the commit, it is done.